lib/serverless.js

```diff
@@ -51,7 +53,9 @@ exports.Plugin = class {
 
         const { servicePath } = this.sls.config;
 
-        const server = this.server = await internals.getServer(servicePath);
+        const rootOrPath = Path.join(servicePath, Hoek.reach(this.sls, 'service.custom.lalalambda.serverPath') || '');
```

Comment:
Consider swapping `Path.join()` with `Path.resolve()` to get absolute path support for free :)

Reply:
I think in this case, using `Path.resolve()` does not gain us anything since `servicePath` is already absolute.  But I changed it because I think this aligns with how you would do it.